lms/templates/pdf_viewer.html

```diff
@@ -114,9 +120,9 @@
               <span data-l10n-id="download_label">Download</span>
             </button>
 
-            <a href="#" id="secondaryViewBookmark" class="secondaryToolbarButton bookmark visibleSmallView" title="Current view (copy or open in new window)" data-l10n-id="bookmark">
+            <button id="secondaryViewBookmark" class="secondaryToolbarButton bookmark visibleSmallView" title="Current view (copy or open in new window)" data-l10n-id="bookmark">
```

Comment:
Note that this is largely a vendor file, and we don't know how the JavaScript is handling this action. I'm having trouble verifying whether or not the functionality of this (and the item below) was broken. I would recommend not changing this file if at all possible.